File: Programming Basics with JS/03. Conditional Statements Advanced - Lab/fruitShop.js
function solve(arg, arg2, arg3) {
    let fruit = String(arg);
    let day = String(arg2);
    let amount = Number(arg3);
    let price = 0;

    if (day != "Monday" && day != "Tuesday" && day != "Wednesday" && day != "Thursday" && day != "Friday" && day != "Saturday" && day != "Sunday") {
        console.log("error");
    } else {
        switch (fruit) {
            case "banana":
                if (day != "Saturday" && day != "Sunday") {
                    price = 2.5 * amount;
                } else {
                    price = 2.7 * amount;
                }
                console.log(price.toFixed(2));
                break;
            case "apple":
                if (day != "Saturday" && day != "Sunday") {
                    price = 1.2 * amount;
                } else {
                    price = 1.25 * amount;
                }
                console.log(price.toFixed(2));
                break;
            case "orange":
                if (day != "Saturday" && day != "Sunday") {
                    price = 0.85 * amount;
                } else {
                    price = 0.9 * amount;
                }
                console.log(price.toFixed(2));
                break;
            case "grapefruit":
                if (day != "Saturday" && day != "Sunday") {
                    price = 1.45 * amount;
                } else {
                    price = 1.6 * amount;
                }
                console.log(price.toFixed(2));
                break;
            case "kiwi":
                if (day != "Saturday" && day != "Sunday") {
                    price = 2.7 * amount;
                } else {
                    price = 3 * amount;
                }
                console.log(price.toFixed(2));
                break;
            case "pineapple":
                if (day != "Saturday" && day != "Sunday") {
                    price = 5.5 * amount;
                } else {
                    price = 5.6 * amount;
                }
                console.log(price.toFixed(2));
                break;
            case "grapes":
                if (day != "Saturday" && day != "Sunday") {
                    price = 3.85 * amount;
                } else {
                    price = 4.2 * amount;
                }
                console.log(price.toFixed(2));
                break;
            default: {
                console.log("error");
                break;
            }
        }
    }


}

solve("orange",
    "Sunday",
    "3")

File: Programming Basics with Java/03. Conditional Statements Advanced - Lab/src/FruitShop.java
import java.util.Scanner;

public class FruitShop {
    public static void main(String[] args) {
        Scanner scan = new Scanner (System.in);

        String item = scan.nextLine();
        String day = scan.nextLine();
        double amount = Double.parseDouble(scan.nextLine());
        double price = 0;
        boolean isError = false;

        if(day.equals("Monday") || day.equals("Tuesday") || day.equals("Wednesday") || day.equals("Thursday") || day.equals("Friday")){
            switch (item){
                case "banana":
                    price = amount * 2.5;
                    break;
                case "apple":
                    price = amount * 1.2;
                    break;
                case "kiwi":
                    price = amount * 2.7;
                    break;
                case "orange":
                    price = amount * 0.85;
                    break;
                case "grapefruit":
                    price = amount * 1.45;
                    break;
                case "pineapple":
                    price = amount * 5.50;
                    break;
                case "grapes" :
                    price = amount * 3.85;
                    break;
                default:
                    isError = true;
                    System.out.println("error");
                    break;
            }
        } else if (day.equals("Saturday") || day.equals("Sunday")){
            switch (item){
                case "banana":
                    price = amount * 2.7;
                    break;
                case "apple":
                    price = amount * 1.25;
                    break;
                case "kiwi":
                    price = amount * 3;
                    break;
                case "orange":
                    price = amount * 0.9;
                    break;
                case "grapefruit":
                    price = amount * 1.6;
                    break;
                case "pineapple":
                    price = amount * 5.6;
                    break;
                case "grapes" :
                    price = amount * 4.2;
                    break;
                default:
                    isError = true;
                    System.out.println("error");
                    break;
            }
        } else {
            isError = true;
            System.out.println("error");
        }

        if(isError == false){
            System.out.printf("%.2f", price);
        }

    }
}
